rc/base/spell.kak

```diff
@@ -9,6 +9,10 @@ def -params ..1 spell -docstring "Check spelling of the current buffer with aspe
         printf %s\\n "set buffer spell_tmp_file ${file}"
     }
     %sh{
+        _echo() {
```

Comment:
Not a big fan on introducing this function, I think I prefer seeing printf direct.y